client/src/App.js

```diff
@@ -20,7 +20,7 @@ const App = () => {
     email: "",
     password: "",
     _id: "609cb2b88951a63fccdcf790",
-    authId: "db3edf52-df87-45b9-9ea5-cd3a38cc2807",
+    authId: "aa94729e-866b-47b1-8c7a-ede2cfd25e51",
```

Comment:
I think I brought this up during a recent meeting, so it might be changed already, but I think we want authID and _id to be dynamically assigned in the final product.

Reply:
Yes! Most definitely - this was just put in before we had the login function pushed so I needed to hard-code credentials in order to interact w the backend.